ai_edge_torch/generative/examples/experimental/gemma/convert_to_tflite.py

```diff
@@ -34,17 +34,17 @@ def convert_gemma_to_tflite(
     quantize: bool = True,
 ):
   """An example method for converting a Gemma 2B model to multi-signature
-  tflite model.
 
+  tflite model.
   Args:
       checkpoint_path (str): The filepath to the model checkpoint, or directory
         holding the checkpoint.
       prefill_seq_len (int, optional): The maximum size of prefill input tensor.
         Defaults to 512.
       kv_cache_max_len (int, optional): The maximum size of KV cache buffer,
         including both prefill and decode. Defaults to 1024.
-      quantize (bool, optional): Whether the model should be quanized.
-        Defaults to True.
+      quantize (bool, optional): Whether the model should be quanized. Defaults
+        to True.
```

Comment:
the formatting seems a bit weird, why is it breaking to two lines?

Reply:
These formattings are done by PyFormat which is used for the code in g3.

As you can see on the left, it was already 2 lines before. This just moved "Defaults" one line up and had to move the rest down to fit under 80 characters.